homeassistant/components/climate/generic_thermostat.py

```diff
@@ -40,7 +40,7 @@
 CONF_COLD_TOLERANCE = 'cold_tolerance'
 CONF_HOT_TOLERANCE = 'hot_tolerance'
 CONF_KEEP_ALIVE = 'keep_alive'
-
+CONF_INITIAL_OPERATION_MODE = 'initial_operation_mode'
```

Comment:
Should we name it `operation_mode`, without initial?
The target temp is also just an initial value

Reply:
See conversation on #10636